backend/src/main/scala/cromwell/backend/standard/callcaching/RootWorkflowFileHashCacheActor.scala

```diff
@@ -73,11 +73,13 @@ class RootWorkflowFileHashCacheActor private(override val ioActor: ActorRef, wor
 
   // Invoke the supplied block on the happy path, handle unexpected states for IoSuccess and IoFailure with common code.
   private def handleHashResult(ioAck: IoAck[_], fileHashContext: FileHashContext)
-                              (notifyRequestersAndCacheValue: NonEmptyList[FileHashRequester] => Unit): Unit = {
+                              (notifyRequestersAndCacheValue: List[FileHashRequester] => Unit): Unit = {
     cache.get(fileHashContext.file) match {
-      case FileHashValueRequested(requesters) => notifyRequestersAndCacheValue(requesters)
+      case FileHashValueRequested(requesters) => notifyRequestersAndCacheValue(requesters.toList)
       case FileHashValueNotRequested =>
-        log.error(s"Programmer error! Not expecting message type ${ioAck.getClass.getSimpleName} with no requesters for the hash: $fileHashContext")
+        log.info(s"Got reply from IoActor for the file ${fileHashContext.file} after the timeout. " +
```

Comment:
```suggestion
        log.info(s"Received a hash value for the file ${fileHashContext.file} with no requesters. This may be due to a benign race condition between hash value receipt and timeout. " +
```